src/modules/ekf2/ekf2_main.cpp

```diff
@@ -439,7 +443,9 @@ Ekf2::Ekf2():
 	_K_pstatic_coef_y(this, "PCOEF_Y"),
 	_K_pstatic_coef_z(this, "PCOEF_Z"),
 	// non EKF2 parameters
-	_airspeed_disabled(this, "FW_ARSP_MODE", false)
+	_airspeed_disabled(this, "FW_ARSP_MODE", false),
+	_land_tgt_static(this, "LTEST_MODE", false)
```

Comment:
Can we do this without having EKF2 read the external parameter directly? The module is optional so we can't depend on this parameter being present.

Reply:
Done